README.md

```diff
@@ -1,7 +1,7 @@
 # Magic Mirror Module: mmm-weatherchart
 This [MagicMirror2](https://github.com/MichMich/MagicMirror) module allows you to show a weather diagram provided by http://www.yr.no
 
-![Screenshot](Screenshot.png "Screenshot")
+![Screenshot](Screenshot1.png "Screenshot")
```

Comment:
Is there a reason for renaming that file here?

Reply:
I saw that github did not update README preview when I replaced your screenshot (cache bug or something, not really sure), thats why I renamed it.